server/modules/category/categoryValidator.js

```diff
@@ -183,7 +183,11 @@ const validateItems = async (category, itemsByParentUuid) => {
     visitedUuids.add(itemUuid)
   }
 
-  return Validation.newInstance(R.isEmpty(itemsValidationsByUuid), itemsValidationsByUuid)
+  return R.isEmpty(itemsValidationsByUuid)
+    ? null
+    : Validation.newInstance(false, { ...itemsValidationsByUuid }, [
```

Comment:
why false?

Reply:
if there is at least one item validation object, then the items are invalid (only validation for invalid items is kept).
If the items are valid, we can return just null, otherwise a Validation object with valid set to false.

Reply:
thanks!